src/locales/en-us/messages-word-list.json

```diff
@@ -124,5 +124,15 @@
         "description": "Placeholder for filter selection",
         "component": "WordFilterPanel",
         "params": ["lang"]
+    },
+    "WORDLIST_DOWNLOAD_BUTTON": {
+        "message": "Download",
+        "description": "Download list button",
+        "component": "WordFilterPanel"
+    },
+    "WORDLIST_DOWNLOAD_NOTICE": {
+        "message": "Some download message",
```

Comment:
How about "Your wordlist will be downloaded as tabular data which can be imported into a spreadsheet program. Columns are separated by semi-colons. (This feature may not work on iOS devices.)"  And maybe we could make the warning about iOS only included if the platform is mobile?

Reply:
@irina060981 I will go ahead and make this change and merge for testing.  I'm not going to worry about making the iOS part optional for now. I am guessing I will be asked to change this text anyway...